### PR TITLE
Replace the `django.utils.timezone.utc` by `datetime.timezone.utc`

### PR DESCRIPTION
**Pull request by hramezani**
fixes https://github.com/jazzband/django-auditlog/issues/427

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 - feat: Display the field name as it would be displayed in Django Admin or use `mapping_field` if available [#428](https://github.com/jazzband/django-auditlog/pull/428)
 - feat: New context manager `disable_auditlog` to turn off logging and a new setting `AUDITLOG_DISABLE_ON_RAW_SAVE`
   to disable it during raw-save operations like loaddata. [#446](https://github.com/jazzband/django-auditlog/pull/446)
+-feat: Replace the `django.utils.timezone.utc` by `datetime.timezone.utc`.  [#448](https://github.com/jazzband/django-auditlog/pull/448)
```
Comment (hramezani) [inline CHANGELOG.md:9]:
```suggestion
-feat: Replace the `django.utils.timezone.utc` by `datetime.timezone.utc`. [#448](https://github.com/jazzband/django-auditlog/pull/448)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,7 @@
 - feat: New context manager `disable_auditlog` to turn off logging and a new setting `AUDITLOG_DISABLE_ON_RAW_SAVE`
   to disable it during raw-save operations like loaddata. [#446](https://github.com/jazzband/django-auditlog/pull/446)
 - Python: Confirm Python 3.11 support ([#447](https://github.com/jazzband/django-auditlog/pull/447))
+- feat: Replace the `django.utils.timezone.utc` by `datetime.timezone.utc`.  [#448](https://github.com/jazzband/django-auditlog/pull/448)
```
Comment (alieh-rymasheuski) [inline CHANGELOG.md:10]: This line still has double space between `[`.

**Review by alieh-rymasheuski — DISMISSED** [review]

File: auditlog_tests/tests.py [hunk 3]
```diff
@@ -634,7 +634,7 @@ def tearDown(self):
         super().tearDown()
 
     def test_model_with_same_time(self):
-        timestamp = datetime.datetime(2017, 1, 10, 12, 0, tzinfo=timezone.utc)
+        timestamp = datetime.datetime(2017, 1, 10, 12, 0, tzinfo=datetime.timezone.utc)
```
Comment (alieh-rymasheuski) [inline auditlog_tests/tests.py:637]: We do `from datetime import timezone` everywhere else in the code, but in the tests we do just `import datetime`. Should we align the style to the former?

File: CHANGELOG.md (hunk 2 above)
Comment (hramezani) [reply CHANGELOG.md]: sorry. got overridden by my force push. Fixed again!

Comment (hramezani) [reply CHANGELOG.md]: sorry. got overridden by my force push. Fixed again!

File: auditlog_tests/tests.py (hunk 3 above)
Comment (hramezani) [reply auditlog_tests/tests.py]: Yes, I've changed it to `from datetime import timezone`

**Comment by hramezani** [comment]
@alieh-rymasheuski I've addressed your comments. Please take another look if you have time.

**Review by alieh-rymasheuski — APPROVED** [review]
@hramezani, I added another stylistic commit on top of your branch.

**Comment by hramezani** [comment]
> @hramezani, I added another stylistic commit on top of your branch.

Thanks @alieh-rymasheuski :+1:

**Pull request merged**